### ISSUE TITLE
frameData is undefined

### ISSUE THREAD

**Issue by seungjulee** [wevm/frog-archived#513]
### Describe the bug

The doc below states that we should be able to parse all values parsed from POST request to the frame, but I'm getting undefined for `frameData`. I want to fetch the logged in user's fid.

https://frog.fm/reference/frog-frame-context#framedata

```
const { frameData } = c
console.log('C FRAME DATA', c)

 const { castId, fid, messageHash, network, timestamp, url } = frameData
```

```
C FRAME DATA undefined
```

### Frog Version

0.18.0

### Check existing issues

- [X] I checked there isn't [already an issue](https://github.com/wevm/frog/issues) for the bug I encountered.

**Comment by dalechyn** [comment]
Are you sure you aren't trying to fetch the fid from initial frame?

Can you check what `c.status` says? It should not state `initial` – as if it is, you can't theoretically get any frameData on GET request frame interaction (the initial one)

**Comment by seungjulee** [comment]
i am fetching one from `app.frame()`. So can I only fetch the user info only after the user clicks a button? is that considered the second render? 

```
app.frame('/miniapp/channel/:channel_id', async (c) => {
  console.log(c)
  
{
  buttonIndex: undefined,
  buttonValue: undefined,
  cycle: 'main',
  deriveState: [Function: deriveState],
  env: {},
  error: [Function: error],
  frameData: undefined,
  initialPath: '/api/miniapp/channel/truther',
  inputText: undefined,
  previousButtonValues: undefined,
  previousState: undefined,
  req: HonoRequest {
    routeIndex: 1,
    bodyCache: { json: [Promise] },
    cachedBody: [Function (anonymous)],
    raw: NextRequest [Request] {
      [Symbol(realm)]: [Object],
      [Symbol(state)]: [Object],
      [Symbol(signal)]: [AbortSignal],
      [Symbol(abortController)]: AbortController {},
      [Symbol(headers)]: [_HeadersList],
      [Symbol(internal request)]: [Object]
    },
    path: '/api/miniapp/channel/truther'
  },
  res: [Function: res],
  status: 'initial',
  transactionId: undefined,
  url: 'http://localhost:3000/api/miniapp/channel/truther',
  var: {},
  verified: false
}

```

**Comment by dalechyn** [comment]
> So can I only fetch the user info only after the user clicks a button? is that considered the second render?

Yes, absolutely!

**Comment by dalechyn** [comment]
I will consider this issue to be resolved, if there are any other questions to that matter, please comment below or open a new issue.
